requirements.txt

```diff
@@ -1,8 +1,23 @@
 bleach==3.0.2
-Markdown==3.0.1
-bleach_whitelist==0.0.9
-Django>=2.1.6
-django_mptt==0.9.1
-djangorestframework>=3.9.1
+bleach-whitelist==0.0.9
+certifi==2019.11.28
+chardet==3.0.4
+defusedxml==0.6.0
+Django==2.2
```

Comment:
I think it would be okay to get any 2.X version `>=2.2,<3.0`

Reply:
yes because django==3.0.0 delete six package and this will causes some problems